install/resources/mysql_db_tables.php

```diff
@@ -724,9 +724,11 @@
   nopermission tinyint(1) NOT NULL default '0',
   location1 int(10) unsigned NOT NULL default '0',
   location2 int(10) unsigned NOT NULL default '0',
+  isunique tinyint(1) NOT NULL default '0',
   PRIMARY KEY(sid),
   KEY location (location1, location2),
   KEY time (time),
+  KEY isunique (isunique),
```

Comment:
It looks like only the MySQL structure adds this key - is this on purpose?

Reply:
Yes, it looks like MyBB doesn't create performance-related indexes for SQLite and PostgreSQL.